src/renderers/webgl-fallback/WebGLBackend.js

```diff
@@ -1571,6 +1571,26 @@ class WebGLBackend extends Backend {
 
 	}
 
+
+	_bindBufferRange( binding, offset, size ) {
+
+		const { gl } = this;
+		const bindingData = this.get( binding );
+		const index = bindingData.index;
+
+		if ( binding.isUniformsGroup || binding.isUniformBuffer ) {
+
+			gl.bindBufferRange( gl.UNIFORM_BUFFER, index, bindingData.bufferGPU, offset, size );
+
+		} else {
+
+			console.warn( 'bindBufferRange is only applicable for uniform buffers' );
```

Comment:
It would be good to always add the component to the message that logs it. E.g.:
```js
console.warn( 'THREE.WebGLBackend: bindBufferRange() is only applicable for uniform buffers.' );
```

Reply:
Oops, actually `_bindBufferRange` was a method I forgot to cleanup that I will work on later on, also I don't think usually warning internally is useful. Thanks for the heads up!